src/exportDataAction/ExportDataAction.js

```diff
@@ -66,8 +69,27 @@ class ExportDataAction {
     }
   }
 
+  historicalFilterString() {
+    const sessionFilter = this.sessionService.getHistoricalSessionFilter();
+
+    if (!sessionFilter) {
+      return '';
+    }
+
+    const forFilename = true;
```

Comment:
this seems unnecessary to name the boolean arg (although i prefer functions to not be extended using boolean flags moreso)